test/AuthorizationHeaderTest.php

```diff
@@ -104,34 +112,54 @@ public function testCaseInsensitiveRequestMethod()
 
     /**
      * Ensures an exception is thrown if a request does not have an Authorization header.
-     *
-     * @expectedException \Acquia\Hmac\Exception\MalformedRequestException
      */
     public function testCreateFromRequestNoAuthorizationHeader()
     {
+        $this->setExpectedException(
```

Comment:
No, it's not necessary, I was trying with a different approach at first, then just copy-paste :)
So I'll revert these to annotations.